components/storage/blockdevice/COMPONENT_I2CEE/I2CEEBlockDevice.cpp

```diff
@@ -22,17 +22,21 @@ using namespace mbed;
 
 I2CEEBlockDevice::I2CEEBlockDevice(
     PinName sda, PinName scl, uint8_t addr,
-    bd_size_t size, bd_size_t block, int freq)
-    : _i2c_addr(addr), _size(size), _block(block)
+    bd_size_t size, bd_size_t block, int freq,
```

Comment:
This doesn't directly relate to this PR, but this driver should now really have a constructor that takes `const i2c_pinmap_t &` and passes to `I2C` to offer the possibility to avoid pulling in pinmap tables.

It would be nice to add this, but I guess as a follow-up PR.

Reply:
Yes, I think this should be a separate PR as it actually has nothing to do with the 8bit-mode.